modules/apps/foundation/configuration-admin/configuration-admin-action/bnd.bnd

```diff
@@ -0,0 +1,6 @@
+Bundle-Name: Liferay Configuration Admin Action
```

Comment:
I have to take that back after seeing all commits.

You are going to have to move these classes to portal-configuration-persistence